### ISSUE TITLE
maxRetries and non-existing screenshots

### ISSUE THREAD

**Issue by Perni1984** [wKich/creevey#182]
We are using creevey in our ci pipeline runs together with a `maxRetries: 6` setting. 

Whenever we are creating a new story and thus the reference screenshot is not existing yet, we are getting back a `Expected image does not exists` message, which is fine. Together with a `maxRetries: 6` setting however, it is retried up to 6 times to find the reference screenshot. I think those screenshots that are not found should not be retried, despite a `maxRetries` setting in `creevey.config.js`.

**Comment by wKich** [comment]
@Perni1984 Yeah, that sounds reasonable. I'll try to take it somewhere in October. Sorry for the delay, I have issues with moving to a new flat and I'm preparing to speak at FrontendConf https://frontendconf.ru/moscow/2021/abstracts/7436
